charts/consul/values.yaml

```diff
@@ -2753,6 +2753,25 @@ meshGateway:
     # DNS entry to point to your mesh gateways.
     static: ""
 
+  # This configures the [`PodDisruptionBudget`](https://kubernetes.io/docs/tasks/run-application/configure-pdb/)
+  # for the service mesh gateway.
+  disruptionBudget:
+    # This will enable/disable registering a PodDisruptionBudget for the
+    # mesh gateway. If this is enabled, it will only register the budget so long as
+    # the mesh gateway.
+    enabled: true
+
+    # The maximum number of unavailable pods. By default, minAvailable will be set to 1
+    # If you need to set this to some value, you will need to add a
+    # --set 'meshGateway.disruptionBudget.maxUnavailable=0'` flag to the helm chart installation
+    # @type: integer
+    maxUnavailable: null
+
+    # The minimum number of available pods.
+    # Takes precedence over maxUnavailable if set.
+    # @type: integer
+    minAvailable: null
```

Comment:
I don't see this value actually being consumed anywhere. I'd expect we either:
1. allow the user to set `minAvailable` for every gateway type, or
2. never allow the user to set `minAvailable` for any gateway type